src/main/java/com/example/demo/repository/BaseTeamRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.example.demo.repository;
+
+import com.example.demo.model.Team;
+
+import java.util.List;
+
+public interface BaseTeamRepository {
```

Comment:
Nice design. This will help to migrate the storage from file to database later on